src/components/MovieList.js

```diff
@@ -2,71 +2,69 @@ import React from 'react';
 
 import MovieListItem from './MovieListItem';
 
-import {SORTING_BY_RATING, SORTING_BY_DATE} from "../constants";
+import { SORTING_BY_RATING, SORTING_BY_DATE } from '../constants';
 
 export default class MovieList extends React.PureComponent {
+  constructor(props) {
+    super(props);
 
-    constructor(props) {
-        super(props);
+    this.state = {
+      cachedProps: null,
+      isEmpty: !(this.props.movieList && this.props.movieList.length),
+      movieList: [],
+    };
+  }
 
-        this.state = {
-            cachedProps: null,
-            isEmpty: !(this.props.movieList && this.props.movieList.length),
-            movieList: []
-        };
+  static getDerivedStateFromProps(nextProps, prevState) {
+    if (prevState.cachedProps === nextProps) {
+      return null;
     }
 
-    static getDerivedStateFromProps(nextProps, prevState) {
-        if (prevState.cachedProps === nextProps) {
-            return null;
-        }
-
-        return {
-            cachedProps: nextProps,
-            movieList: MovieList.sortList(nextProps.sortingType, nextProps.movieList),
-            isEmpty: !(
-                nextProps.movieList && nextProps.movieList.length
-            )
-        };
-    }
+    return {
+      cachedProps: nextProps,
+      movieList: MovieList.sortList(nextProps.sortingType, nextProps.movieList),
+      isEmpty: !(
+        nextProps.movieList && nextProps.movieList.length
+      ),
+    };
+  }
 
     static compareDates = (movie_1, movie_2) => {
-        const date_1 = String(movie_1.year).split('-').join('');
-        const date_2 = String(movie_2.year).split('-').join('');
+      const date_1 = String(movie_1.year).split('-').join('');
+      const date_2 = String(movie_2.year).split('-').join('');
 
-        return Number(date_2) - Number(date_1);
+      return Number(date_2) - Number(date_1);
     };
 
     static compareRating = (movie_1, movie_2) => (
-        Number(movie_2.rating) - Number(movie_1.rating)
+      Number(movie_2.rating) - Number(movie_1.rating)
     );
 
     static sortList(sortingType, list) {
-        list = [...list];
-
-        switch (sortingType) {
+      list = [...list];
```

Comment:
It's a bad practice to reassign function arguments inside of the function, please create a new variable instead